### PR TITLE
Add iceberg-light colorscheme

### PR DESCRIPTION
**Pull request by ufUNnxagpM**
Adds the complementary light theme. I've based all of the colors off the upstream at https://github.com/cocopon/iceberg.vim/blob/master/autoload/iceberg/palette/light.vim, but there are a few things that I didn't find equivalents to

* experimental colors
* lualine terminal, inactive, command

## How to use

Use by setting `colorscheme=iceberg-light`

### CODE REVIEW

File: lua/lualine/themes/iceberg.lua [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        replace = {
+            a = { bg = "#c57339", fg = "#e8e9ec" },
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        command = {
+            a = { bg = "#a093c7", fg = "#e8e9ec" },
```
Comment (oahlen) [inline lua/lualine/themes/iceberg.lua]: Use purple from light theme
```suggestion
            a = { bg = "#7759b4", fg = "#e8e9ec" },
```

File: lua/lualine/themes/iceberg.lua [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        terminal = {
+            a = { bg = "#89b8c2", fg = "#e8e9ec" },
```
Comment (oahlen) [inline lua/lualine/themes/iceberg.lua]: Use cyan from light theme
```suggestion
            a = { bg = "#3f83a6", fg = "#e8e9ec" },
```

File: iceberg-light.toml [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+pale   = "hsl($hue_pale, 0.30, 0.45)"
+
+# normal
+normal_bg = "hsl($hue_base, 0.10, 0.92)"
+normal_fg = "hsl($hue_base, 0.20, 0.25)"
+
+# TODO experimental colors
+pink = "hsl(330, 0.32, 0.68)"
+teal = "hsl(165, 0.32, 0.65)"
+
```
Comment (oahlen) [inline iceberg-light.toml]: These were unused experimental colors of the dark variant so they are redundant here

**Review by oahlen — CHANGES_REQUESTED** [review]
Very nice job! 😀

I have been thinking about porting the light variant so this is appreciated.
A couple of colors in the lualine section could be adresed before merging.

File: iceberg-light.toml (hunk 3 above)
Comment (ufUNnxagpM) [reply iceberg-light.toml]: gotcha, just removed these

**Review by oahlen — APPROVED** [review]

**Pull request merged**